### ISSUE TITLE
Viewing secrets does not work anymore

### ISSUE THREAD

**Issue by passw0rd123** [jhaals/yopass#17]
The new restructured version seems to be broken if its installed manually or by docker.

Due to the missing updated installation steps I tried to follow the steps in the Dockerfile, but I always end with an 404 when Itry to view the secret. Also the full secret link is shown in the server logs now...

If I use the docker containers I end up with the same results...

```
2018/03/02 14:59:30 Starting yopass. Listening on port 1337
127.0.0.1 - - [02/Mar/2018:14:59:36 +0100] "GET /result HTTP/1.0" 404 19
127.0.0.1 - - [02/Mar/2018:14:59:57 +0100] "GET / HTTP/1.0" 304 0
127.0.0.1 - - [02/Mar/2018:14:59:57 +0100] "GET /static/css/main.6f022e07.css HTTP/1.0" 304 0
127.0.0.1 - - [02/Mar/2018:14:59:57 +0100] "GET /static/js/main.09471f93.js HTTP/1.0" 304 0
127.0.0.1 - - [02/Mar/2018:15:00:12 +0100] "POST /secret HTTP/1.0" 200 50
127.0.0.1 - - [02/Mar/2018:15:00:30 +0100] "GET /s/a1c5ddef-59c4-4615-963b-49f758ca3fdf/C0atSB6B1IqquLuh HTTP/1.0" 404 19
```

**Comment by jhaals** [comment]
Thanks for reporting this!
So the new website is a single page application in react meaning all paths not covered by the API should be routed to index.html and be handled by the router in react, looks like it's not happening in some browsers, Chrome and Firefox seems to handle it just fine but not Safari, is that the browser you're using? Need to figure out if gorilla mux has some custom 404 logic so those requests can be routed to public/index.html instead. The other alternative is to go back to `/#/path` which isn't as nice.

**Comment by jhaals** [comment]
fixed in 5.0.2
